### DISCUSSION TITLE
import boa

### DISCUSSION THREAD

**Discussion by fabianestebanmolina** [Cyfrin/moccasin-full-course-cu#10] [General]
hi everyone , i have a problem and don't know what's going on i'm trying to do the python import videos on your site... i have this error on my terminal 

import boa
boa.eval("empty(uint256)")

/bin/python3 /home/fabian/Escritorio/learntoken/my_project/fuelboa?.py
Traceback (most recent call last):
  File "/home/fabian/Escritorio/learntoken/my_project/fuelboa?.py", line 1, in <module>
    import boa
ModuleNotFoundError: No module named 'boa'

i already did de pip install 
this is my version of python 3.12.3
this my version of vyper 0.4.0+commit.e9db8d9
and my version of titanoboa Version: 0.2.5

**Comment by iburr** [comment]
@fabianestebanmolina I would highly suggest trying to use UV, it makes syncing dependencies with your project so easy.

**Reply by iburr** [reply]
Also make sure that your on that specfic version of python in your interpreter.

**Reply by EngrPips** [reply]
> @fabianestebanmolina I would highly suggest trying to use UV, it makes syncing dependencies with your project so easy.

I see.

**Reply by Scofield-Idehen** [reply]
Check if you are installing in a Venv or global environment. If you are sure where you are, then install the library again. 

Using UV install might work if you already have those dependencies, so check again by running `which uv`. If it's there, then you can use it; otherwise, you have to install again. 

Patrick explored Venv so you can go back to that course and check again.

**Reply by EngrPips** [reply]
Great heads-up.
